src/openrct2/actions/RideDemolishAction.hpp

```diff
@@ -217,4 +218,93 @@ struct RideDemolishAction : public GameActionBase<GAME_COMMAND_DEMOLISH_RIDE, Ga
 
         return res;
     }
+
+private:
+
+    money32 MazeRemoveTrack(uint16 x, uint16 y, uint16 z, uint8 direction) const
+    {
+        auto setMazeTrack = MazeSetTrackAction(x, y, z, direction, _rideIndex, GC_SET_MAZE_TRACK_FILL);
+        setMazeTrack.SetFlags(GetFlags());
+
+        auto queryRes = setMazeTrack.Query();
+        if (queryRes->Error == GA_ERROR::OK)
+        {
+            auto execRes = setMazeTrack.Execute();
+            if (execRes->Error == GA_ERROR::OK)
+            {
+                return execRes->Cost;
+            }
+        }
+
+        return MONEY16_UNDEFINED;
```

Comment:
MONEY32_UNDEFINED

Reply:
God damnit, why are there two, I keep messing this up.